ssv/types.go

```diff
@@ -35,10 +36,12 @@ type AttesterCalls interface {
 
 // ProposerCalls interface has all block proposer duty specific calls
 type ProposerCalls interface {
+	// SubmitValidatorRegistration submits a validator registration
+	SubmitValidatorRegistration(pubkey []byte, feeRecipient bellatrix.ExecutionAddress, sig phase0.BLSSignature) error
 	// GetBeaconBlock returns beacon block by the given slot and committee index
-	GetBeaconBlock(slot phase0.Slot, committeeIndex phase0.CommitteeIndex, graffiti, randao []byte) (ssz.Marshaler, spec.DataVersion, error)
```

Comment:
what is the reason for deletion?

Reply:
According to beacon node spec https://ethereum.github.io/beacon-APIs/#/Validator/produceBlockV2, we don't need committee index, we need only slot, graffiti, and randao

Reply:
ok so there was a difference between our spec and impl?
because it is hard for me to beleieve that the impl had the same mistake

Can you also fix the comment?